classes/email-templates/class-pmpro-email-template-membership-expired.php

```diff
@@ -138,9 +138,10 @@ public static function get_email_template_variables_with_description() {
 	 */
 	public function get_email_template_variables() {
 		global $wpdb;
+		$membership_level_id = $this->membership_level_id;
```

Comment:
This line is what fixes the issue, I wanted to share this as this PR now does a bit of refactoring and I myself was confused to where the actual fix was.